web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -802,7 +811,7 @@ const TeamReviews = ({ onBack, periodId }) => {
   const approveReviewAssignment = async (assignment, approved) => {
     try {
       const res = await resolve({
-        method: 'PUT',
+        method: assignment.id === null ? 'POST' : 'PUT',
```

Comment:
When the assignment does not yet have an `id`, we need to send a POST request instead of a PUT.